windows-driver-docs-pr/develop/removing-coinstallers.md

```diff
@@ -0,0 +1,81 @@
+  
+For settings and configuration that cannot be set within the driver itself, a driver package may also include user-mode runtime components that alter the settings and configuration. This can be a user-facing app or a Win32 service that updates the configuration. If a persistent component such as a service is used, ensure that its functionality is necessary and cannot be performed in a less resource-intensive manner, such as within a driver package INF or within the driver itself. Great care should be taken to ensure that any service is only running when the relevant devices are connected (see [Service Triggers](../../windows/win32/services/service-trigger-events.md), [Win32 services interacting with devices](../install/best-practices-win32services-interacting-with-devices.md), and [Registering for device interface notifications](../drivers/install/registering-for-notification-of-device-interface-arrival-and-device-removal.md)), and the service meets the latest requirements (for instance, passing [API Validator](./validating-windows-drivers.md#apivalidator)).
+
+## Installing Associated Software
+The 'Componentized' portion of the [DCH driver requirements](../develop/dch-principles-best-practices.md) introduced a concept called the SoftwareComponent, which is a mechanism to decouple the install of a device driver from it associated software. When a software component is created by the INF, it will automatically create a child device that maps to the software component. This child device will exist for the purpose of installing the software associated to the parent device. This software can be installed and updated independently of the main device and driver.
+
+Within a SoftwareComponent driver package INF, the recommended mechanism to install software is using an AddSoftware directive. This will trigger the download an installation of software from the Windows Store.  
+  
+[SoftwareComponent](../install/using-a-component-inf-file.md)  
+[AddSoftware](../install/inf-addsoftware-directive.md)  
+[DCH Driver Package Sample](./dch-example.md)  
+[Pairing a driver with a UWP app](../install/pairing-app-and-driver-versions.md)
+
+## Dependencies across drivers and devices
+### Device start/enumeration ordering dependencies
+To the extent possible, inter-device dependencies or start ordering requirements should be avoided.
+
+For ACPI-enumerated devices, the dependency object (_DEP) may be used in the ACPI firmware to enforce device start ordering. Alternatively, the device stack can be created in such a way that there is a custom bus driver that enumerates the children in dependency order. 
```

Comment:
```suggestion
For ACPI-enumerated devices, the dependency object (_DEP) may be used in the ACPI firmware to enforce device start ordering. Alternatively, the device stack can be created in such a way that there is a custom bus driver that enumerates the child devices in dependency order. 
```

Reply:
@zlockard do you have any comments on Jason's point?

Reply:
My intent was to suggest various solutions that might make this transition seem doable. I don't have strong opinions on this in particular